lib/auth0/api/v2/organizations.rb

```diff
@@ -0,0 +1,332 @@
+        # @return [json] All Organizations
+        def organizations(options = {})
+          request_params = {
+            per_page:       options.fetch(:per_page, nil),
+            page:           options.fetch(:page, nil),
+            include_totals: options.fetch(:include_totals, nil)
+          }
+          get(organizations_path, request_params)
+        end
+        alias get_organizations organizations
+
+        # Create a new organization.
+        # @see ttps://auth0.com/docs/api/management/v2/#!/Organizations/post_organizations
+        # @param options [hash] See https://auth0.com/docs/api/management/v2/#!/Organizations/post_organizations for available options
+        # @return [json] Returns the created organization.
+        def create_organization(options = {})
+          post(organizations_path, options)
+        end
+
+        # Get an organization by id. A token with read:organizations scope is required
+        # @see https://auth0.com/docs/api/management/v2/#!/Organizations/get_organizations_by_id
+        # @param organization_id [string] The organization_id of the user to retrieve.
+        #
+        # @return [json] Returns the organization with the given organization_id if it exists.
+        def organization(organization_id)
+          raise Auth0::MissingOrganizationId, 'Must supply a valid organization_id' if organization_id.to_s.empty?
+          path = "#{organizations_path}/#{organization_id}"
+          get(path)
+        end
+
+        # Get an organization by name. A token with read:organizations scope is required.
+        # @see https://auth0.com/docs/api/management/v2/#!/Organizations/get_name_by_name
+        # @param organization_id [string] The Organization ID
+        #
+        # @return [json] Returns the organization with the given organization_id if it exists.
+        def organization_by_name(organization_name)
+          raise Auth0::MissingOrganizationId, 'Must supply a valid organization_name' if organization_name.to_s.empty?
+          path = "#{organizations_path}/name/#{organization_name}"
+          get(path)
+        end
+
+
+        # Deletes a single user given its id
+        # @see https://auth0.com/docs/api/management/v2/#!/Organizations/delete_organizations_by_id
+        # @param organization_id [string] The Organization ID
+        def delete_organization(organization_id)
+          raise Auth0::MissingOrganizationId, 'Must supply a valid organization_id' if organization_id.to_s.empty?
+          path = "#{organizations_path}/#{organization_id}"
+          delete(path)
+        end
+
+        # Update an existing organization.
+        # @see https://auth0.com/docs/api/management/v2/#!/Organizations/patch_organizations_by_id
+        # @param organization_id [string] The Organization ID
+        # @param body [hash] The optional parameters to update.
+        #
+        # @return [json] Returns the updated user.
+        def patch_organization(organization_id, body)
```

Comment:
What's the convention for PATCH methods in the library?

Just noticed that we have `create_organization` for POSTs but here we're using the verb in the name. Wouldn't this be `update_organization`?

Fair enough if that's the convention elsewhere, just struck me as odd.